woocommerce/woocommerce-android
# Implement first draft of "Connect to card reader" screen

Parent issues #3725

This PR adds a first draft of the UI necessary to start connecting to a card reader.


https://user-images.githubusercontent.com/2722505/118505536-7ff87280-b6fa-11eb-830c-5b9ecc6b69d0.mov

To test

* HW reader is not supported for this flow -> simulated reader will be used by default
* Testing site with WCPay installed and activated is required
* Download CI build or checkout the branch, build and run
* Log into a store.
* Navigate to Settings > Manage card reader .
* Tap "redirect to cardreaderdetailfragment"
* Notice the new screen.

<img src="https://user-images.githubusercontent.com/2722505/118505590-8c7ccb00-b6fa-11eb-8da7-f5f462ab25c4.png" width="350"/>

* Tap the Connect Card reader button. 
* A modal popover should be triggered:

 <img src="https://user-images.githubusercontent.com/2722505/118505762-b6ce8880-b6fa-11eb-8607-7e474d4e7cf5.png" width="350"/>


Update release notes:
